Dockerfile-centos.template

```diff
@@ -5,13 +5,13 @@ ARG R_VERSION=%%R_VERSION%%
 ARG OS_IDENTIFIER=%%OS_IDENTIFIER%%
 
 # Install R
-RUN wget https://cdn.posit.co/r/${OS_IDENTIFIER}/pkgs/R-${R_VERSION}-1-1.x86_64.rpm && \
+RUN wget "https://cdn.posit.co/r/${OS_IDENTIFIER}/pkgs/R-${R_VERSION}-1-1.$(arch).rpm" && \
```

Comment:
where do we define `arch`?

Reply:
`arch` is a command that returns either `x86_64` or `aarch64` on Linux:
```sh
$ arch
x86_64
```
It might be on macOS too. But we're using this in most of our multiarch docker images to dynamically get the arch.